### DISCUSSION TITLE
Don't show 404 urls in Top Pages

### DISCUSSION THREAD

**Discussion by danirabbit** [plausible/analytics#200] [Anything Else]
It looks like we have someone abusing our public dashboard by spamming `/theirblogurl.com`

![Screenshot from 2020-07-01 11 37 34@2x](https://user-images.githubusercontent.com/7277719/86279869-97d11180-bb8f-11ea-815c-390edd4a2660.png)

**Comment by ukutaht** [comment]
Right, makes sense. In #96 we've been asked to show 404s in a separate list.

Would make sense to split them out somehow so by default you see found pages but can choose to only show 404s or other non-2xx status codes.

**Comment by danirabbit** [comment]
Yeah hiding behind a filter/tab would also be a great solution! Thanks for responding quickly

**Comment by cassidyjames** [comment]
Agreed, splitting them out would also be handy for knowing what old/dead links people are stumbling upon, to set up redirects on our site.
